events/syscall/unexpected_k8s_nodeport_connection.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package syscall
+
+import (
+	"github.com/falcosecurity/event-generator/events"
+	"os/exec"
+	"strconv"
+)
+
+var _ = events.Register(
+	UnexpectedK8sNodePortConnection,
+	events.WithDisabled(), // this rule is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func UnexpectedK8sNodePortConnection(h events.Helper) error {
+	if h.InContainer() {
+		port := 31000
+
+		hostIP, err := getHostEth0IP()
+		if err != nil {
+			return err
+		}
+		path, err := exec.LookPath("nc")
+		if err != nil {
+			// If we don't have an netcat, just bail
+			return &events.ErrSkipped{
+				Reason: "netcat utility not found in path",
+			}
+		}
+		cmd := exec.Command(path, hostIP, strconv.Itoa(port), "<", "/dev/null")
```

Comment:
```suggestion
		cmd := exec.Command(path, hostIP, strconv.Itoa(port), "<", "/dev/null")
```

Can the same result be achieved just using go code (without using an external tool)? :thinking:

Reply:
sure @leogr let me try..

Reply:
I changed it to use UDP connection from go/net package instead of netcat. Please take a look.